packages/wallet/backend/src/quote/service.ts

```diff
@@ -51,14 +56,40 @@ export class QuoteService implements IQuoteService {
       throw new BadRequest('Not enough funds in account')
     }
 
-    const asset = await this.deps.rafikiClient.getAssetById(assetId)
+    let asset: Asset | undefined = await this.deps.rafikiClient.getAssetById(
+      assetId
+    )
     if (!asset) {
       throw new NotFound()
     }
 
-    const value = BigInt((params.amount * 10 ** asset.scale).toFixed())
+    const originalValue = BigInt((params.amount * 10 ** asset.scale).toFixed())
+    let value = originalValue
 
     let paymentUrl = params.receiver
+
+    const pp = await this.deps.paymentPointerService.getExternalPaymentPointer(
```

Comment:
to be consistent with namings, we should name this paymentPointer, I also tend to use quickly pp, but if you check above we have existingPaymentPointer and other explicit long names, so it makes sense to have paymentPointer here as well.